Website/routes/login.js

```diff
@@ -13,6 +13,27 @@ router.get('/login', function(req, res, next) {
 		secure: req.session.secure,
 	}, req.savedContext));
 });
+
+router.get('/loginUserInsecure', async function(req, res, next) {
```

Comment:
Rather than duplicating the route code into the two request types, I think it would be more simple/clean to create one async function and pass it to both of them:
var loginUser = async func...

route.get('/loginUserInsecure',loginUser)
route.post('/loginUser',loginUser)

Reply:
You're right. The secondary function came early in development when I planned to take this idea in a different direction, but with what I settled on the duplicated code isn't necessary. I'll clean it up and simplify.

Reply:
Added